scripts/ci/test

```diff
@@ -10,6 +10,8 @@ if  [[ ! -z "${TRAVIS_TAG}" ]]; then
     exit 0
 fi
 
+[[ "$(python --version | cut -d' ' -f2)" == "${PYTHON:-3.7}"* ]]
+
```

Comment:
What does this actually do? 
```
bash-3.2$ [[ "foo" == "bar"* ]]
bash-3.2$ [[ "foo" == "foo"* ]]
bash-3.2$
```

Reply:
Oh right, Travis will bail when $? is not zero

Reply:
Yeah, it's a bit too much magic, but for now I just wanted to make sure what is being used, because I was almost convinced to merge this PR.